src/test/java/com/hocs/test/glue/CreateCaseStepDefs.java

```diff
@@ -86,8 +89,40 @@ public void aCaseIsCreated() {
         page.sleep(500);
     }
 
-    @When("^I create a case with <Topic>$")
-    public void aCaseWithSpecificTopicIsCreated() {
+    @When("^I create a \"([^\"]*)\" case with \"([^\"]*)\"$")
+    public void aCaseWithSpecificTopicIsCreated(String caseType, String topic) {
+        switch (caseType.toUpperCase()) {
+            case "DCU MIN":
+                createCase.createDCUMinSingleCase();
+                setSessionVariable("caseType").to(caseType);
+                dataInput.dataInputFullFlow();
+                topics.fromMarkupStartSelectATopic(topic);
+                break;
+            default:
```

Comment:
We've started using `pendingStep(pageName + " is not defined within " + getMethodName());` to replace `assumeNoteNull`. It will cause your test to drop out with a user defined exception. In this case `pageName + " is not defined within " + getMethodName())`.

I've added the first change as suggested commits (you might want to squash them into one commit).